src/OpenLoco/src/Objects/ObjectManager.cpp

```diff
@@ -930,10 +932,54 @@ namespace OpenLoco::ObjectManager
     }
 
     // 0x004796A9
-    void updateYearly1()
+    void updateDefaultLevelCrossingType()
```

Comment:
You've already indicated the ObjectManager isn't a great place for this. Perhaps the Scenario namespace would be a better fit?